src/main/java/frc/robot/Constants.java

```diff
@@ -58,29 +59,57 @@ public static final class PivotConstants {
     /**  */
     public static final double gearRatio = 93.3333333;
     /**  */
-    public static final ArmFeedforward pivotFFModel = new ArmFeedforward(
-      0.35, 
-      0.35, 
-      1.79, 
-      0.3);
-    /**  */
-    public static final PIDController pivotPIDController = new PIDController(
-      12, 
-      0, 
-      0.00);
-    /**  */
-    public static final TrapezoidProfile pivotProfile = new TrapezoidProfile(new TrapezoidProfile.Constraints(
-      2,
-      5));
-    /** Ramp Rate of the pivot System ID in volts per second */
-    public static final double pivotSysIdRampRate = 0.2;
-    /** Setp Voltage of the pivot System ID in volts */
-    public static final double pivotSysIdStepVolt = 7;
-    /** Timeout of the pivot System ID in volts */
-    public static final double pivotSysIdTimeout = 30;
+
+    public static final TrapezoidProfile.Constraints pivotConstraints = 
+      new TrapezoidProfile.Constraints(
+        Units.radiansToDegrees(2), 
+        Units.radiansToDegrees(5));
+
+    public static final ProfiledPIDController pivotPIDController = 
```

Comment:
I assume you didn't intend to change these values, but I would recommend not touching these for now. Only touch the characterization routines